src/Settings.cpp

```diff
@@ -152,6 +152,9 @@ QVariant Settings::getDefaultValue(const QString& group, const QString& name)
     if(group == "General" && name == "toolbarStyle")
         return static_cast<int>(Qt::ToolButtonTextBesideIcon);
 
+    if(group == "General" && name == "DBFileExtensions")
+        return QObject::tr("SQLite database files (*.db *.sqlite *.sqlite3 *.db3);;All files (*)");
```

Comment:
When always adding the ```All files``` option in the ```FileDialog::getSqlDatabaseFileFilter()``` function as I suggested above, it would obviously need to be removed here :smile:

Reply:
;)